modules/custom/adobe_analytics/js/analytics-footer.js

```diff
@@ -0,0 +1 @@
+_satellite.pageBottom()
```

Comment:
I agree. But this is how it is requested in the instructions by Service Canada, as an inline javascript tag. But I put it in a file since it is very discouraged to add inline js in drupal. My assumption is that it would serve as an indicator of the end on the page scope somewhere in the custom js script that is loaded inside the <head> tag